extensions/wikia/MyHome/ActivityFeedTag.php

```diff
@@ -53,6 +51,13 @@ function ActivityFeedTag_render($content, $attributes, $parser, $frame) {
 		)
 	);
 
+	$style = getStyle($parameters);
+
 	wfProfileOut(__METHOD__);
 	return "<div$style>$feedHTML</div>$snippets";
```

Comment:
Personally, I am not a fan of such interpolations. I would sacrifice performance a bit and use `sprintf()` and use it consistently elsewhere (e.g. in the `getStyle()` method, where string concatenation with `.` is used.